dcrwallet.go

```diff
@@ -188,6 +188,7 @@ func startPromptPass(w *wallet.Wallet) {
 			"This will complete syncing of the wallet accounts " +
 			"and then leave your wallet unlocked. You may relock " +
 			"wallet after by calling 'walletlock' through the RPC.")
+		backendLog.Flush()
```

Comment:
I would move this to before where the actual prompt occurs.  The prompt is run in a loop and code after the prompt in that same loop may write to the logger, so it needs to be flushed again.